schemas/test/en/test_list_change_evaluates_sections.json

```diff
@@ -27,6 +27,18 @@
     },
     "sections": [
         {
+            "summary": {
```

Comment:
All sections should now have summaries now, this one didn't. So it was removed the test as you can now just can't skip through a section not changing anything when complete or add a summary to allow a value to be changed

Reply:
Agreed, this pattern doesn't really lend itself to social current, and I do wonder about relationships as they naturally don't have summaries at present.